build-tools/timing/timing.csproj

```diff
@@ -6,23 +6,24 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
   </PropertyGroup>
   <Import Project="..\..\Configuration.props" />
+  <Import Project="timing.targets" />
```

Comment:
This file should probably follow `build-tools/dependencies/dependencies.csproj` in spirit and:

1.  `<Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />`
2. Override/set `$(BuildDependsOn)`.

That way the `@(ProjectReference)` values will actually *mean something*. (Without a call to the `ResolveReferences` target, `@(ProjectReference)` is meaningless.